ta/os_test/os_test.c

```diff
@@ -35,6 +35,7 @@
 #include "os_test.h"
 #include "testframework.h"
 #include "test_float_subj.h"
+#include "undef_instr.h"
```

Comment:
Generally I try to avoid inline assembly unless there's some benefit. In this case there isn't except that we can avoid a couple of files.

If you'd rather have it as inline assembly I'll change it, I hope we're not starting a new trend though.